src/content/resources/concepts/layer/assessment/assessment-plan/_index.md

```diff
@@ -13,7 +13,7 @@ aliases:
 
 | Assessment Plan Schema | Assessment Plan Converters | Reference
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/json-index/) |
 | [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_assessment-plan_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/assessment-plan/xml-index/) |
```

Comment:
"How do I use this?"  link https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml  is broken

Reply:
I figured out how I missed this, I rebased the half of links that were incorrect (I completed the XML to JSON link replacement, but not the JSON to XML ones you cited). I rebased the original commit in the series and this was addressed.